### DISCUSSION TITLE
Need help with recovering a saved session

### DISCUSSION THREAD

**Discussion by xodotLBO** [gioxx/MarvellousSuspender#212] [Q&A]
So I have my old windows files and I saved my session before I re-installed windows (didn't EXPORT it, just SAVED it) people told me you can recover that saved session and export/import into new. Anyone know how or where its located (appdata?)

**Comment by vondeliusc** [comment]
I also had Windows problems and had to restore an image and lost about 3 weeks of saved sessions.
I just need to know where the saved sessions are stored.

**Comment by gioxx** [comment]
I am only responding now, it is probably completely unnecessary and I apologize.
I don't know if, at the time, you had tried looking at the old TGS recovery procedure (also valid for TMS, just by changing extension IDs): https://github.com/greatsuspender/thegreatsuspender/issues/526

**Comment by xodotLBO** [comment]
Considering this was 2 years ago, I probably did. Please try to make a firefox version, ty
